### DISCUSSION TITLE
ln -s /mnt/c/Users/user /home/user

### DISCUSSION THREAD

**Discussion by devy1540** [microsoft/WSL#9151] [General]
can i try to make symbolic link

`ln -s /mnt/c/Users/user /home/user`

is there any problem?

**Comment by okibcn** [comment]
Try it and let us know. In any case, you can directly define the user folder to /mnt/c/Users/user Learn how here: https://www.howtouselinux.com/post/linux-command-change-user-home-directory-in-linux

**Comment by NotTheDr01ds** [comment]
Better for a Discussion than an Issue.

But no, I don't recommend this.  There will be multiple problems:

* Performance:  Accessing files on the Windows drive is currently *much* slower than the ext4 filesystem.  See #4197.
* Permissions:  Can probably be worked around, but you'll at least need to add the [metadata option](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#automount-options) to the automount options using `/etc/wsl.conf`.
* Other POSIX incompatibilities:  IIRC, the file locking mechanism expected from a POSIX drive isn't going to work with the 9P network filesystem, but I could be wrong on this.

**Reply by devy1540** [reply]
I wanted to use just one home directory in Windows, so I left the following question.

**Comment by devy1540** [comment]
@okibcn @NotTheDr01ds 

I tried something like this

```
ln -s /mnt/c/Users/test /home/test
sudo vim /etc/passwd
/home/user -> /home/test
```

It works, but it's slow.
